src/plugins/dashboard/public/application/dashboard_app_controller.tsx

```diff
@@ -1109,12 +1111,6 @@ export class DashboardAppController {
         $scope.model.filters = filterManager.getFilters();
         $scope.model.query = queryStringManager.getQuery();
         dashboardStateManager.applyFilters($scope.model.query, $scope.model.filters);
-        if (dashboardContainer) {
-          dashboardContainer.updateInput({
```

Comment:
This is not needed. After removing all dashboard container updates are handled in single place right now.
Looking forward to this file being refactored.

Reply:
I'm in the middle of completely re-doing this file as part of deangularization. It will be a functional component - I may end up pinging you for some input at some point.

Reply:
@ThomThomson, that's great news 👍 Will be glad to chime in. 
From perspective of this pr important bit is that we are calling `dashboardContainer.updateInput` in a single place